packages/msked/test/index.ts

```diff
@@ -0,0 +1,300 @@
+import test from 'blue-tape'
+import { applyMask, getPositionInMasked, getPositionInValue, isLongerThanMask } from '..'
+
+test('getPositionInMasked', (t) => {
```

Comment:
so far we have a single stream of Tape test results for entire metarepo so it's a good idea to prefix tests like `msked: getPositionInMasked`

Reply:
Will do